addons/main/initKeybinds.sqf

```diff
@@ -4,7 +4,7 @@
     // Clear brush if possible
     if (_player call FUNC(canClearBrush)) then {
         _player call FUNC(clearBrush);
-    };
 
-    true
+        true
```

Comment:
Ah good call. The `true` should just go away right? it doesn't do anything here

Reply:
I'm having doubts - shouldn't the `if` statement return `true` (if condition is met), which makes the entire code block return `true`?

Reply:
the `if` as you have it doesn't return anything due to closing `;`

Reply:
The intention behind the `true` in the `if` block was to intercept the keystroke if the action was successful, but I imagine it shouldn't pose a problem if the `true` is removed altogether.

Reply:
Yeah that's what I mean. Regardless of clearing brush or not, it shouldn't block any following keybindings.